tensorflow/core/util/mkl_util.h

```diff
@@ -130,6 +130,17 @@ enum class MklQuantization {
 
 static const int kSmallBatchSize = 32;
 
+#ifdef ENABLE_MKLDNN_V1
+inline void execute_primitives(
```

Comment:
I just realized that there is already an existing, similar function `ExecutePrimitive` (also in this file -- line 681). Maybe add a TODO to merge them if possible?

Reply:
@penpornk added the TODO, yes, you are right, this should be merged in the feature code clean. Thanks.